lib/app/global/widgets/app_bar/components/components_app_bar.dart

```diff
@@ -16,13 +15,11 @@ class LogoAppBar extends StatelessWidget {
     return MouseRegion(
       cursor: SystemMouseCursors.click,
       child: GestureDetector(
-        child: Image.asset(
-          "assets/images/logo.png",
-          width: screen.width * 0.2,
-        ),
-        onTap: () => Navigator.push(
-            context, MaterialPageRoute(builder: (context) => const HomePage())),
-      ),
+          child: Image.asset(
+            "assets/images/logo.png",
+            width: screen.width * 0.2,
+          ),
+          onTap: () => Modular.to.pushReplacementNamed('/')),
```

Comment:
Olá, notei que vocês utilizaram Modular.to.pushReplacementNamed('/')), para substituir a implementação incorreta do navigator na appbar. 

Utilizar pushReplacementNamed causa crashes durante o hard reset e a atualização via f5. Basta alterar para Modular.to.navigate('/')), que resolve âmbos os problemas e evitará conflitos no merge para a release. Grato!

Reply:
Feito